sources/javascripts/concat/base.js

```diff
@@ -537,6 +537,38 @@
     });
   };
 
+  // ===========================================================================
+  // Change product image position on narrower screens (mobile devices)
+  // ===========================================================================
+
+  var handleProductPageContent = function () {
+    $(document).ready(function () {
+      changeProductImagePos();
+    });
+
+    $(window).resize(debounce(function () {
+      changeProductImagePos();
+    }, 25));
+
+    var changeProductImagePos = function () {
+      var productGallery = $('.js-product-gallery');
+      var productImageContentBox = $('.js-content-item-box');
+      var buyBtnContent = $('.js-buy-btn-content');
+
+      if ($('.js-buy-btn-content .edy-buy-button-container').length >= 1) {
+        if ($(window).width() <= 640) {
+          if ($('.js-buy-btn-content + .js-product-gallery').length <= 0) {
```

Comment:
It seems that the `length` cant be under 0 in any way, so the check should be like this:
```js
if ($('.js-buy-btn-content + .js-product-gallery').length === 0) { ... }
```